.github/pull-request-template.md

```diff
@@ -0,0 +1,16 @@
+<!-- Refer to CONTRIBUTING.md for more details and examples.
+https://github.com/prometheus/client_golang/blob/main/CONTRIBUTING.md#how-to-fill-the-pr-template  
+-->
+### Describe your PR
+
+
+### What type of PR is this?
+
+<!-- Format: /kind followed by ONE of the type {fix, bugfix, enhancement, feature, feat, change, release-note-none} -->
```

Comment:
Can we give example?

Reply:
There are examples in contributing guide, but here putting one example would be nice.

Reply:
I think people will eventually look at previously merged PRs when looking for examples.

If we make the template too long, it'll get ignored 😅

Reply:
We previously discussed about this. Initially, examples were included, but they made the template excessively long. Additionally, the regex used in the [release-notes](https://github.com/kubernetes/release/tree/master/cmd/release-notes) tool, which we are using internally, was incorrectly capturing content from the examples. Therefore, we decided to remove the examples to keep the PR template concise and easy to read.